recipe_sha: 2c80ae1dc661d267ec064ec7973f7f67d14d0427064d85173a21e9e602393cc0  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  2024-12-13T05:04:36 noarch Writing current_repodata subset
  2024-12-13T05:04:36 noarch Writing index HTML
  /opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py:501: UserWarning: RECIPE_PATH received is a file (/opt/recipe/meta.yaml).
  It should be a path to a folder.
  Forcing conda-build to use the recipe file.
    warnings.warn(
  WARNING: No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.22
  Adding in variants from internal_defaults
  Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  Attempting to finalize metadata for boms
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  BUILD START: ['boms-1.1.0-py39he88f293_2.tar.bz2']
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_h_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/boms_1734066277590/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol


  The following NEW packages will be INSTALLED:

      _libgcc_mutex:      0.1-conda_forge              conda-forge
      _openmp_mutex:      4.5-2_kmp_llvm               conda-forge
      brotli-python:      1.1.0-py39hf88036b_2         conda-forge
      bzip2:              1.0.8-h4bc722e_7             conda-forge
      ca-certificates:    2024.8.30-hbcca054_0         conda-forge
      certifi:            2024.8.30-pyhd8ed1ab_0       conda-forge
      cffi:               1.17.1-py39h15c3d72_0        conda-forge
      charset-normalizer: 3.4.0-pyhd8ed1ab_1           conda-forge
      h2:                 4.1.0-pyhd8ed1ab_1           conda-forge
      hpack:              4.0.0-pyhd8ed1ab_1           conda-forge
      hyperframe:         6.0.1-pyhd8ed1ab_1           conda-forge
      idna:               3.10-pyhd8ed1ab_1            conda-forge
      ld_impl_linux-64:   2.43-h712a8e2_2              conda-forge
      libffi:             3.4.2-h7f98852_5             conda-forge
      libgcc:             14.2.0-h77fa898_1            conda-forge
      libgcc-ng:          14.2.0-h69a702a_1            conda-forge
      libhwloc:           2.11.2-default_h0d58e46_1001 conda-forge
      libiconv:           1.17-hd590300_2              conda-forge
      liblzma:            5.6.3-hb9d3cd8_1             conda-forge
      libnsl:             2.0.1-hd590300_0             conda-forge
      libsqlite:          3.47.2-hee588c1_0            conda-forge
      libstdcxx:          14.2.0-hc0a3c3a_1            conda-forge
      libstdcxx-ng:       14.2.0-h4852527_1            conda-forge
      libuuid:            2.38.1-h0b41bf4_0            conda-forge
      libxcrypt:          4.4.36-hd590300_1            conda-forge
      libxml2:            2.13.5-h0d44e9d_1            conda-forge
      libzlib:            1.3.1-hb9d3cd8_2             conda-forge
      llvm-openmp:        19.1.5-h024ca30_0            conda-forge
      mkl:                2024.2.2-ha957f24_16         conda-forge
      mkl-service:        2.4.2-py39h0296b93_0         conda-forge
      ncurses:            6.5-he02047a_1               conda-forge
      openssl:            3.4.0-hb9d3cd8_0             conda-forge
      pip:                24.3.1-pyh8b19718_0          conda-forge
      pybind11:           2.13.6-pyh1ec8472_2          conda-forge
      pybind11-global:    2.13.6-pyh415d2e4_2          conda-forge
      pycparser:          2.22-pyh29332c3_1            conda-forge
      pysocks:            1.7.1-pyha55dd90_7           conda-forge
      python:             3.9.21-h9c0c6dc_1_cpython    conda-forge
      python_abi:         3.9-5_cp39                   conda-forge
      readline:           8.2-h8228510_1               conda-forge
      requests:           2.32.3-pyhd8ed1ab_1          conda-forge
      setuptools:         75.6.0-pyhff2d567_1          conda-forge
      tbb:                2021.13.0-hceb3a55_1         conda-forge
      tk:                 8.6.13-noxft_h4845f30_101    conda-forge
      tzdata:             2024b-hc8b5060_0             conda-forge
      urllib3:            2.2.3-pyhd8ed1ab_1           conda-forge
      wheel:              0.45.1-pyhd8ed1ab_1          conda-forge
      zstandard:          0.23.0-py39h08a7858_1        conda-forge
      zstd:               1.5.6-ha6fb4c9_0             conda-forge

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 589, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 209, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3712, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2439, in build
      create_build_envs(top_level_pkg, notest)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2213, in create_build_envs
      environ.create_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1050, in create_env
      _execute_actions(prefix, precs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1338, in _execute_actions
      progressive_fetch_extract.execute()
    File "/opt/conda/lib/python3.10/site-packages/conda/core/package_cache_data.py", line 900, in execute
      raise CondaMultiError(not_cancelled)
  conda.CondaMultiError: Error with archive /opt/conda/pkgs/mkl-2024.2.2-ha957f24_16.conda.  You probably need to delete and re-download or re-create this file.  Message was:

  failed with error: [Errno 28] No space left on device
# Last 100 lines of the build log.
